process/banning.html

```diff
@@ -44,37 +44,48 @@ <h4>Nearby</h4>
-        <li>If anyone raises an issue to the CEO that a participant fails to meet the requirements of <a href="https://www.w3.org/Consortium/Process/#ParticipationCriteria">Individual Participation Criteria</a>, then the CEO is empowered to investigate. If the participant indeed is failing to meet those requirements, then after at least one warning and at least one subsequent violation, the CEO may temporarily or permanently suspend participation.</li>
-      </ul>
-<p>4. In extreme cases (such as death threats, or physical threats in general), the CEO and COO, acting together, may suspend or remove someone from participation in all groups. The CEO and COO must announce the action to W3C Management along with the rationale.</p>
-
-<p>5. Typically a suspension will encompass participation in all forms, including meetings, email, IRC, Github posting, etc. In no case may a person be blocked from seeing public information (e.g. mailing list or Github posts, public minutes).</p>
-
-<p>6. No individual may be banned from sending email to the W3C Ombuds team. (Bots may be banned.)</p>
-
-<p>7. The W3C Systems team may block spammers and others who abuse our systems.</p>
-
-<p>8. The W3C Systems team will provide technical support to implement suspensions.</p>
-
+</li>
+<li><p>In case of disagreement between the W3C group chair and team contact, each should explain their reasons to the CEO.</p></li>
+<li>
+  <p>Following violation of the <a href="https://www.w3.org/Consortium/cepc/">W3C Code of Ethics and Professional Conduct</a>, at least one warning by the chair(s), and at least one subsequent violation of the CEPC, a W3C group chair and team contact, acting together, may:</p>
+  <ol>
+<li>
+  <p>temporarily or permanently suspend participation by a person if the person has not formally joined the group.</p>
+</li>
+<li>
+  <p>temporarily suspend participation for up to 5 business days if the person is a participant the group after emailing the CEO (and the Member's Advisory Committee representative if the participant is not an Invited Expert).  Any subsequent suspension within 6 months would need to be requested from the CEO.</p>
+</li>
+<li>propose to suspend or remove a participant from a single group - typically for egregious and repetitive CEPC violations - by making a request to the CEO. If the participant is a W3C Member representative, the CEO will coordinate the response with the Member's Advisory Committee representative.</li>
+</li>
+</ol>
+</li>
+<li>If anyone raises an issue to the CEO that a participant fails to meet the requirements of <a href="https://www.w3.org/Consortium/Process/#ParticipationCriteria">Individual Participation Criteria</a>, then the CEO is empowered to investigate. If the participant indeed is failing to meet those requirements, then after at least one warning and at least one subsequent violation, the CEO may temporarily or permanently suspend participation.</li>
```

Comment:
This "raises an issue to the CEO" seems to run against the CEPC's section on [Reporting](https://www.w3.org/Consortium/cepc/#Reporting), which suggests getting assistance from an Ombudsperson or senior W3C Management.